components/eventing-controller/reconciler/subscription-nats/reconciler.go

```diff
@@ -27,7 +27,7 @@ import (
 type Reconciler struct {
 	client.Client
 	cache.Cache
-	natsClient       *handlers.Nats
+	natsClient       handlers.MessagingBackendHandler
```

Comment:
`s/natsClient/messagingBackend/`

or even shorter

`s/natsClient/backend/`

Reply:
```suggestion
	backend       handlers.MessagingBackendHandler
```
just to make it easier to apply :D

Reply:
Done